Category-Colors/category-colors
# Feature/style components

# Description
Issue # (add the issue number here, if applicable)

Describe what you're adding
Added styling to the panel box per the wireframe
# QA
List out how to test your changes and add a screenshot of what it should look like, when applicable.

See the screenshot attached, as of now, I'm testing the changing of the background when toggling it, still need to tweak the drop-down arrow to point to the left and bottom when opened.

![toggle](https://github.com/Category-Colors/category-colors/assets/57161327/3d86a3ca-ed4d-4a0c-951f-cc61fadc3c5b)
